stylus/base/defaults.styl

```diff
@@ -9,3 +9,13 @@ $error
     border-radius   4px
     background-color red
     color            white
+.coz-hidden
```

Comment:
Yes, i was wondering about utility classes like this one. In my head, @extend modules are for functionnal modules (like $error, $dialog, etc) and this utility class isn't a module, it's more like a tool to easily and quickly enhance you view.
With this in mind i was thinking if maybe make it an @extend module wasn't in some way overkill. But i may be too cautious 👅